orion/sites/admin/src/announce/announce.js

```diff
@@ -5,9 +5,62 @@ import ReactDOM from 'react-dom';
 
 export class AnnouncePage extends React.Component {
 	render() {
+		var numAnnouncements = 3; 	/* this needs to be a variable later */
+
+		/* these fake announcements will eventually be replaced with real announcements */
+		let fakeAnnounceA = {
+			date: "02/15/2020",
+			author: "Melon King",
+			message: "Only evil people waste watermelon"
+		};
+		let fakeAnnounceB = {
+			date: "02/16/2020",
+			author: "Fruit Hater",
+			message: "Watermelons are fruit, therefore they deserve to dry on the vine"
+		};
+		/* will use this to test text wrapping later */
+		let fakeAnnounceC = {
+			date: "02/23/2020",
+			author: "Talkative",
+			message: "This will be a longer message later"
+		};
 		return (
       <div id="view-announce">
-        <h1>All Announcements</h1>
+        <h1>All Announcements ({numAnnouncements}) </h1>
+				<ul> {/* this is the subheading */}
+					<div className="announce-lists">
```

Comment:
Here, announce-lists is a `div` while the one below are `li`. Try to make them consistent. It will make your life easier in the future. Either way is fine, but as long as it is consistent. Here, maybe I'd lean to go towards with making "announce-lists" to `li`. And that's because it's the child of a `ul` parent.

Then, the children of "announce-lists" could either by a `ul` followed by many `li`s or simply a couple of `div`s.